tests/validation/tests/v3_api/test_global_role.py

```diff
@@ -0,0 +1,348 @@
+    gr = create_gr()
+    remove_resource(gr)
+    token = rbac_get_user_token_by_role(CLUSTER_MEMBER)
+    validate_delete_global_role(token, gr, False)
+
+
+@if_test_rbac
+def test_rbac_global_role_delete_project_owner(remove_resource):
+    gr = create_gr()
+    remove_resource(gr)
+    token = rbac_get_user_token_by_role(PROJECT_OWNER)
+    validate_delete_global_role(token, gr, False)
+
+
+@if_test_rbac
+def test_rbac_global_role_delete_project_member(remove_resource):
+    gr = create_gr()
+    remove_resource(gr)
+    token = rbac_get_user_token_by_role(PROJECT_MEMBER)
+    validate_delete_global_role(token, gr, False)
+
+
+@if_test_rbac
+def test_rbac_global_role_delete_project_read_only(remove_resource):
+    gr = create_gr()
+    remove_resource(gr)
+    token = rbac_get_user_token_by_role(PROJECT_READ_ONLY)
+    validate_delete_global_role(token, gr, False)
+
+
+@if_test_rbac
+def test_rbac_global_role_edit_cluster_owner(remove_resource):
+    gr = create_gr()
+    remove_resource(gr)
+    token = rbac_get_user_token_by_role(CLUSTER_OWNER)
+    validate_edit_global_role(token, gr, False)
+
+
+@if_test_rbac
+def test_rbac_global_role_edit_cluster_member(remove_resource):
+    gr = create_gr()
+    remove_resource(gr)
+    token = rbac_get_user_token_by_role(CLUSTER_MEMBER)
+    validate_edit_global_role(token, gr, False)
+
+
+@if_test_rbac
+def test_rbac_global_role_edit_project_owner(remove_resource):
+    gr = create_gr()
+    remove_resource(gr)
+    token = rbac_get_user_token_by_role(PROJECT_OWNER)
+    validate_edit_global_role(token, gr, False)
+
+
+@if_test_rbac
+def test_rbac_global_role_edit_project_member(remove_resource):
+    gr = create_gr()
+    remove_resource(gr)
+    token = rbac_get_user_token_by_role(PROJECT_MEMBER)
+    validate_edit_global_role(token, gr, False)
+
+
+@if_test_rbac
+def test_rbac_global_role_edit_project_read_only(remove_resource):
+    gr = create_gr()
+    remove_resource(gr)
+    token = rbac_get_user_token_by_role(PROJECT_READ_ONLY)
+    validate_edit_global_role(token, gr, False)
+
+
+@pytest.fixture(scope='module', autouse="True")
+def create_project_client(request):
+    client, cluster = get_global_admin_client_and_cluster()
+    create_kubeconfig(cluster)
+
+
+def create_gr():
+    return validate_create_global_role(ADMIN_TOKEN, False, True)
+
+
+def generate_template(name, new_user_default=False):
+    template = deepcopy(template_manage_catalog)
+    if new_user_default:
+        template["newUserDefault"] = "true"
+    else:
+        template["newUserDefault"] = "false"
+    if name is None:
+        name = random_name()
+    template["name"] = name
+    return template
+
+
+def validate_create_global_role(token, new_user_default=False,
+                                permission=False):
+    client = get_client_for_token(token)
+    t_name = random_name()
+    template = generate_template(t_name, new_user_default)
+    try:
```

Comment:
Same comment here as in case of create catalog
**Scenario:**
```
User does NOT have permission to create global role
But he somehow is able to.
This will not be caught here as the try block will not raise an error
```